assets/development-plan.csv

```diff
@@ -1,2 +1,2 @@
 reference,name,description,developmentPlanType,periodStartDate,periodEndDate,developmentPlanGeography,documentationUrl,organisations,entryDate,startDate,endDate,timetableEvents
-XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX,Birmingham New Local Plan,We are working on a new Local Plan for Birmingham which will guide how the city will develop in the future and provide policies to guide decisions on development proposals and planning applications up to 2042.,type,05-Jan-23,05-Jan-23,geography,,Birmingham LPA,05-Jan-23,05-Jan-23,05-Jan-23
\ No newline at end of file
+XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX,Birmingham New Local Plan,We are working on a new Local Plan for Birmingham which will guide how the city will develop in the future and provide policies to guide decisions on development proposals and planning applications up to 2042.,type,05-Jan-23,05-Jan-23,geography,,Birmingham LPA,05-Jan-23,05-Jan-23,05-Jan-23,
```

Comment:
I've just noticed, the date formats here are incorrect? E.g. we have `05-Jan-23` instead of `2023-01-05`.

Unrelated, we also don't have the reference (which should be reflected in the timetable example file)

Reply:
I am happy to update those as part of my change :)

Reply:
done :)

Reply:
done! :)